src/main/resources/templates/layout.html

```diff
@@ -59,9 +59,18 @@
         </ul>
     </div>
 </nav>
-<div role="main" layout:fragment="content" id="content" class="container">
-    <!--CONTENT-->
+<div class="container">
+    <div role="main" layout:fragment="content" id="content">
+        <!--CONTENT-->
+    </div>
+
+    <footer class="footer mt-4 mb-3 text-center">
+        <hr/>
```

Comment:
nie lepiej zrobic to cssem i jakis div z border-style-top ?

Reply:
fixed